src/world_interface/noop_world.cpp

```diff
@@ -57,6 +62,8 @@ types::DataPoint<int32_t> getMotorPos(robot::types::motorid_t motor) {
 	return {};
 }
 
+void setMotorVel(int32_t targetVel) {}
```

Comment:
Don't think this is the right signature, it should also have a `motorid_t` parameter.